lib/graphiti/runner.rb

```diff
@@ -1,18 +1,27 @@
 module Graphiti
   class Runner
     attr_reader :params
-    include Graphiti::Base
 
     def initialize(resource_class, params, query = nil)
       @resource_class = resource_class
       @params = params
       @query = query
+
+      validator = RequestValidator.new(jsonapi_resource, params)
+
+      if validator.validate
```

Comment:
maybe this should be `validate!` which raises the error?

Reply:
Added a second `validate!` method to `RequestValidator` which moves this logic there.